src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -358,7 +358,7 @@ private static Binder GetEnclosingBinderInternalWithinRoot(SyntaxNode node, int
                 LocalFunctionSymbol function = GetDeclaredLocalFunction(binder, ownerOfTypeParametersInScope.Identifier);
                 if ((object)function != null)
                 {
-                    binder = function.SignatureBinder;
+                    binder = function.ParameterBinder;
```

Comment:
`SignatureBinder` is used for attribute binding, hence, it doesn't have type parameters in scope. This code path used by `LookupPosition` wants to have type parameters in scope, so I switched to using `ParameterBinder` here.